mathics/core/evaluation.py

```diff
@@ -54,6 +54,38 @@ def _thread_target(request, queue) -> None:
         queue.put((False, exc_info))
 
 
+def kill_thread(thread):
```

Comment:
Small thing if you are up for it -> bool would be nice. Maybe even a type annotation for thread if that is easily accessible.